web/massastation/src/pages/store/MyPlugin.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+
+import {
+  Button,
+  Certificate,
+  MassaWallet,
+  Plugin,
+} from '@massalabs/react-ui-kit';
+import { FiArrowUpRight, FiRefreshCcw, FiTrash2 } from 'react-icons/fi';
+import { IMassaPlugin } from './MyStation';
+
+enum PluginStatus {
+  Up = 'Up',
+  Down = 'Down',
+}
+
+function MyPlugin({ plugin }: { plugin: IMassaPlugin }) {
+  const [myPlugin, setMyPlugin] = useState<IMassaPlugin>(plugin);
+  function handlePluginState(method: string) {
+    // TODO :
+    // const {mutate } = usePost<any>("plugin-manager");
+    // const ok = mutate({method:"${method}"})
+    const newStatus = method === 'start' ? PluginStatus.Up : PluginStatus.Down;
```

Comment:
What is 'start'? Why we rely on a string like that? 🤔

Reply:
this line was for testing purpose I'll remove it 
start/stop/update is the method given in the body https://github.com/massalabs/thyra/blob/027834090096aa978e287acf6ce1700433f454f0/api/swagger/server/restapi/resource/swagger.yml#L618

Reply:
Export constants then with that names:

export const PLUGIN_START = 'start'
export const PLUGIN_STOP = 'stop'
... etc

then use in all places you have

method === PLUGIN_START